src/components/grid-aware/StatsBlock/StatsBlock.tsx

```diff
@@ -32,10 +44,22 @@ const StatsBlock = ({ title, statCards, theme }: StatsBlockProps) => (
     />
     <div className={s.bleedMainContent}>
       <section className={s.gridParent}>
-        <h1 className={s.title}>{title}</h1>
+        <h1
+          className={`${s.title} ${
+            subtitle && titleColor ? s[titleColor] : ""
```

Comment:
You can remove the `subtitle` check here. Since we've made `titleColor` its own property, it would make the most sense to someone using the `StatsBlock` component that `titleColor` will control the color of the title regardless of the presence of a subtitle. Removing the unnecessary complexity makes it more direct and easier to use.